content/fact/securelevel.md

```diff
@@ -0,0 +1,15 @@
+---
+title: securelevel(7)
+---
+
+securelevel is a security mechanism in OpenBSD kernel. The OpenBSD kernel
+ provides four levels of system security, from insecure mode to draconian mode.
+ This ultimate mode is a last line of defence if the superuser account is
+ compromised or limit potential damage.
```

Comment:
Can we add a little more detail or practical examples on what securelevel does? 

Currently it reads like "its security for your system" and thats all. 

What do you think?

Reply:
No idea about this measure.